var/spack/repos/builtin/packages/xz/package.py

```diff
@@ -61,7 +68,69 @@ def determine_version(cls, exe):
         match = re.search(r"xz \(XZ Utils\) (\S+)", output)
         return match.group(1) if match else None
 
+
+class RunAfter(object):
```

Comment:
Nothing inherits this class. FWIW it would never be used on Windows, so you could move this method to the AutotoolsBuilder vs. using inheritance.

Reply:
Moved to autotools builder, thanks!